src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/NuGetVSActionTelemetryService.cs

```diff
@@ -11,19 +10,18 @@ namespace NuGet.VisualStudio
     /// <summary>
     /// Telemetry service class for restore operation
     /// </summary>
-    public class NuGetVSActionTelemetryService : INuGetTelemetryService
+    public class NuGetVSTelemetryService : INuGetTelemetryService
     {
         private ITelemetrySession _telemetrySession;
+        public static readonly Lazy<string> NuGetVersion
```

Comment:
why NuGetVersion is here? I'd imagine it will part of your utility class VSTelemetryServiceUtility instead of being in Service